docs/spec/governance/01_concepts.md

```diff
@@ -64,6 +43,21 @@ governance module (eg. `ParamChangeProposal`), which then execute the respective
 module's proposal handler when a proposal passes. This custom handler may perform
 arbitrary state changes.
 
+## Deposit
+
+To prevent spam, proposals must be submitted with a deposit in a the coins defined in the `MinDeposit` param. The voting period will not start as long as the proposal's deposit is smaller than the `MinDeposit`.
```

Comment:
```suggestion
To prevent spam, proposals must be submitted with a deposit in the coins defined in the `MinDeposit` param. The voting period will not start until the proposal's deposit equals `MinDeposit`.
```